spec/examples/complex.schema.md

```diff
@@ -23,17 +23,17 @@ This is an example schema that uses types defined in other schemas.
 
 # Complex References  Properties
 
-| Property | Type | Required | Defined by |
-|----------|------|----------|------------|
-| [and](#and) | complex | Optional | Complex References  (this schema) |
-| [or](#or) | complex | Optional | Complex References  (this schema) |
-| [refabstract](#refabstract) | `object` | **Required** | Complex References  (this schema) |
-| [reflist](#reflist) | Simple | Optional | Complex References  (this schema) |
-| [refnamed](#refnamed) | Simple | Optional | Complex References  (this schema) |
-| [xor](#xor) | complex | Optional | Complex References  (this schema) |
-| `int.*` | `integer` | Pattern | Complex References  (this schema) |
-| `str.*` | `string` | Pattern | Complex References  (this schema) |
-| `*` | any | Additional | this schema *allows* additional properties |
+| Property | Type | Required | Nullable | Defined by |
+|----------|------|----------|----------|------------|
+| [and](#and) | complex | Optional  | No | Complex References  (this schema) |
+| [or](#or) | complex | Optional  | No | Complex References  (this schema) |
+| [refabstract](#refabstract) | `object` | **Required**  | No | Complex References  (this schema) |
+| [reflist](#reflist) | Simple | Optional  | No | Complex References  (this schema) |
+| [refnamed](#refnamed) | Simple | Optional  | No | Complex References  (this schema) |
+| [xor](#xor) | complex | Optional  | No | Complex References  (this schema) |
+| `int.*` | `integer` | Pattern  No | Complex References  (this schema) |
```

Comment:
@trieloff there seems to be some issue here. No separation between "Pattern" and "No".  Also what does "pattern" mean for `Required` ?

Reply:
Good eye ;) I'll fix it.

"Pattern" in the context of "Required" means that it is a pattern property, i.e. not required, but the constraint applies to all properties that match the pattern.

Reply:
@trieloff Have you fixed this? Then I can approve and merge the PR